lib/ClangImporter/ClangImporter.cpp

```diff
@@ -3262,6 +3262,64 @@ void ClangImporter::Implementation::lookupAllObjCMembers(
   }
 }
 
+bool ClangImporter::Implementation::loadNamedMembers(
+    const Decl *D, DeclName N, uint64_t contextData,
+    TinyPtrVector<ValueDecl *> &Members) {
+
+  auto *DC = dyn_cast<DeclContext>(D);
+  assert(DC && "loadNamedMembers on a Decl that's not a DeclContext");
```

Comment:
This one you should just write `cast`. We're used to that, and you still get the backtrace.

Reply:
fixed.